images/dcos/bootstrap.sh

```diff
@@ -220,7 +220,7 @@ EOF
 #
 # nginx, proxying the apiserver and serving kubectl binaries
 #
-sed "s,<PORT>,${apiserver_proxy_port},;s,<APISERVER>,http://${host_ip}:${apiserver_port}," /etc/nginx/conf.d/default.conf.template > /etc/nginx/conf.d/default.conf
+sed "s,<PORT>,${apiserver_proxy_port},;s,<APISERVER>,https://${host_ip}:${apiserver_secure_port}," /etc/nginx/conf.d/default.conf.template > /etc/nginx/conf.d/default.conf
```

Comment:
in a comment on the related SSL/TLS "investigation" ticket you mentioned that https wasn't required for SPDY upgrade. so why the additional overhead of using https here?